RS/RS-Project-6/NQueensProblem/NQueensProblem.java

```diff
@@ -0,0 +1,70 @@
+class NQueensSolution {
```

Comment:
I think the only classes related to dictionary has to be here in the github. please confirm